### DISCUSSION TITLE
Jenkins & GIT Hub Configuration

### DISCUSSION THREAD

**Discussion by Baharul1** [community/community#22900] [New to GitHub]
<p>Hi All,</p>
<p>I am trying to connect Jenkins with GitHub and getting error as “Failed to connect with repository” in Jenkins.<br>
For this I have followed below steps…</p>
<h1>GitHub</h1>
<ol>
<li>Created repository and From Settings–&gt;IntegrationAnd Services have updated “Jenkins Hook url”.</li>
<li>Tested connectivity and from GitHub and looks fine.</li>
</ol>
<h1>Jenkins</h1>
<ol>
<li>Installed Jenkins on Tomcat.</li>
<li>When tried to create an item with “Free style project” getting below error in connecting with GIT repo in “Source Code management”<br>
“Failed to connect to repository : Error performing command: git.exe ls-remote -h <a href="https://github.com/Baharul1/Try_Jenkins.git" rel="nofollow noopener">https://github.com/Baharul1/Try_Jenkins.git</a> HEAD”<br>
3.Also have verified network connectivity looks good from system where jenkins is insta to GitHub</li>
</ol>
<p>Can anyone please help on this.</p>
<p>Please note I didn’t installed GitHub on local system. Does it require to install always?</p>
<p>Thanks<br>
Baharul Islam</p>

**Comment by nadiajoyce** [comment]
<p>Hi <span class="mention">@Baharu1</span>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You mention that you “didn’t installed GitHub”. I’m assuming that you mean that you didn’t install Git? If you didn’t install Git, that would probably be the reason why Jenkins is having a problem executing <code>git.exe</code> that it mentions in the error message.</p>
<p>I haven’t worked with Jenkins before though, so I can’t be certain.</p>
